docs/cli.md

```diff
@@ -0,0 +1,104 @@
+# CLI
+
+Operates on directories as a unit of work:
+
+```
+my-model/
+  substratus.yaml <---- Reads Dataset/Model/Server/Notebook from here.
+  Dockerfile <--- Optional
+  run.ipynb
+```
+
+## Alternative Names
+
+"AI CLI":
+
+```bash
+# Looks for "ai.yaml"...
+
+ai notebook .
+ai nb .
+ai run .
+```
+
+## Notebook
+
+```bash
+strat nb (notebook)
+```
+
+## List
+
+```bash
+strat ls (list)
+```
+
+```
+strat ls
+
+models/
+  facebook-opt-125m
+  falcon-7b
+
+datasets/
+  squad
+
+servers/
+  falcon-7b
+```
+
+```
+strat ls models
+
+facebook-opt-125m
+falcon-7b
+```
+
+```
+strat ls models/falcon-7b
+
+model.bin
+vocab.json
+```
+
+## Run
+
+* Tar & upload
+* Remote build
+* Wait for Ready
+
+```bash
+strat run .
+```
+
+## View
+
+* Grab `run.html` (converted notebook) and serve on localhost.
```

Comment:
can this be run only once ready or could we get results like these while the model is being ran? if so does it keep on converting the notebook to html every x seconds?

I will probably answer this myself based on this PR review